lib/screens/signup/signup.dart

```diff
@@ -31,11 +31,13 @@ class _SignupScreenState extends State<SignupScreen> {
   }
 
   _updateCountryCode(Locale myLocale) {
-    Map localeData = codes.firstWhere((Map code) => code['code'] == myLocale.countryCode);
-    if (mounted) {
-      setState(() {
-        countryCode = CountryCode(dialCode: localeData['dial_code']);
-      });
+    if (myLocale.countryCode != null) {
+      Map localeData = codes.firstWhere((Map code) => code['code'] == myLocale.countryCode) ?? '‎+1';
```

Comment:
If firstWhere will not find a value it will throw an exception, need to add `orElse` here

Reply:
Yeah, thanks.